hopr/hopr-lib/src/config.rs

```diff
@@ -54,6 +59,11 @@ pub struct Chain {
     #[serde(default = "just_true")]
     #[default = true]
     pub fast_sync: bool,
+    #[serde(default = "just_false")]
+    #[default = false]
+    pub enable_logs_snapshot: bool,
```

Comment:
Why is it still here though? It's not necessary.

Reply:
Agreed to keep it.